### ISSUE TITLE
False positive on indent with try/catch and typealias

### ISSUE THREAD

**Issue by eygraber** [pinterest/ktlint#1788]
## Expected Behavior
The following code is valid:

```
fun test() = try {
  1
}
catch(_: Throwable) {
  2
}
```

and

```
typealias Foo =
  HashMap<Bar, Baz?>
```

## Observed Behavior
ktlint formats the code like this:

```
fun test() = try {
  1
}
  catch(_: Throwable) {
    2
  }
```

and

```
typealias Foo =
HashMap<Bar, Baz?>
```

## Steps to Reproduce
Run `ktlint --format`

.editorconfig:

```
[*.{kt,kts}]
ij_kotlin_allow_trailing_comma = false
ij_kotlin_allow_trailing_comma_on_call_site = false
indent_size = 2
indent_style = space
insert_final_newline = true
ktlint_experimental = enabled
ktlint_experimental_comment-wrapping = disabled
ktlint_experimental_function-naming = disabled
ktlint_experimental_function-signature = disabled
ktlint_experimental_property-naming = disabled
ktlint_standard_annotation = disabled
ktlint_standard_filename = disabled
ktlint_standard_keyword-spacing = disabled
ktlint_standard_max-line-length = disabled
ktlint_standard_no-semi = disabled
ktlint_standard_package-name = disabled
ktlint_standard_spacing-between-declarations-with-annotations = disabled
ktlint_standard_string-template = disabled
max_line_length = 120
```

The formatting is correct when ktlint 0.47.1 is used

## Your Environment
* Version of ktlint used: 0.48.2
* Operating System and version: Ubuntu 22.04
